### DISCUSSION TITLE
中继模式下的问题

### DISCUSSION THREAD

**Discussion by nyt35** [vxiaov/vClash#33] [Q&A] [answered]
TUF-AX3000，华硕KOOLSHARE固件386，vclash在中继模式下貌似不能正常工作，yacd面板没任何连接及流量数据，订阅文件没问题，请问大神如何设置啊？

**Comment by Awkee** [comment, marked as answer]
在这个模式下，防火墙、IP 分享以及 NAT 功能是禁用的。 你应该在外网网关路由器上安装这个插件，中继路由器不用做任何设置。

**Reply by nyt35** [reply]
感谢大神的解答，有一个事不明白，就是openwrt或者padavan，貌似都可以这是中继+ap的模式，这样好像就管事了，但是华硕或者梅林的固件我没看到有这个功能，还是需要单独设置什么？

**Reply by Awkee** [reply]
没实际尝试过，没办法回答。
